client/src/services/tangleCacheService.ts

```diff
@@ -1,10 +1,13 @@
+/* eslint-disable max-len */
+/* eslint-disable camelcase */
```

Comment:
Why did we disable these?

Reply:
camelcase is disabled because of the page_size query property. `Identifier 'page_size' is not in camel case`. That property is transformed to query string using fethcHelper.urlParams.

I will break down the long lines and enabled max-len.